.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] It would be better if we have a descriptive README. Here is a link to the Microvere [README template](https://github.com/microverseinc/readme-template). Without a descriptive README can render our project invalid. However, because this was not mentioned out to you earlier. I would urge you to fix this and keep it in mind

Reply:
Gotcha.